### ISSUE TITLE
💡 idea: Is it possible to make it work properly without setting up the github plugin?

### ISSUE THREAD

**Issue by hkjang** [mattermost/mattermost-plugin-agents#142]
## Description

Is it possible to make it work properly without setting up the github plugin?

I want to make it work properly by setting only the mattermost ai plugin.

**Comment by crspeller** [comment]
@hkjang Yes it should work fine without the Github plugin. Just without the Github functionality. Are you running into some issue?
